tsconfig.json

```diff
@@ -12,24 +12,16 @@
     "importHelpers": true,
     "target": "es2015",
     "allowSyntheticDefaultImports": true,
-    "typeRoots": [
-      "node_modules/@types"
-    ],
-    "lib": [
-      "es2018",
-      "dom"
-    ],
+    "typeRoots": ["node_modules/@types"],
+    "lib": ["es2018", "dom"],
     "paths": {
-      "ng-imgix": [
-        "dist/ng-imgix"
-      ],
-      "ng-imgix/*": [
-        "dist/ng-imgix/*"
-      ]
-    }
+      "ng-imgix": ["projects/ng-imgix/dist"],
+      "ng-imgix/*": ["dist/ng-imgix/*"]
+    },
+    "skipLibCheck": true
```

Comment:
For two reasons:
a) because type-checking the libs (@types/node) etc is just wasted time usually
b) because it can cause type errors when there are conflicting versions of global libs installed. Now you're probably thinking "oh but that's a problem right?" Actually, they're not, because if they caused a type issues in our program it would show up as a type error somewhere in our source code, not in those libs.